omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ObsController1_9Test.java

```diff
@@ -84,6 +87,31 @@ public void shouldGetAll() throws Exception {
 	}
 	
 	@Test
+	public void shouldGetallIncludingVoidedObs() throws Exception {
+		
+		executeDataSet("encounterWithObsGroup1_9.xml");
+		
+		MockHttpServletRequest getAllNonVoidedObsRequest = newGetRequest(getURI());
+		getAllNonVoidedObsRequest.addParameter("encounter", "62967e68-96bb-11e0-8d6b-9b9415a91465");
+		MockHttpServletResponse AllNonVoidedObsResponse = handle(getAllNonVoidedObsRequest);
+		List<Object> AllNonVoidedObsList = deserialize(AllNonVoidedObsResponse).get("results");
+		
+		assertEquals(AllNonVoidedObsList.size(), 6);
+		
+		MockHttpServletRequest deleteRequest = newDeleteRequest(getURI() + "/" + "47f18998-96cc-11e0-8d6b-9b9415a91465");
+		deleteRequest.addParameter("reason", "test voided obs");
+		handle(deleteRequest);
+		
+		MockHttpServletRequest getAllObsIncludingVoidedRequest = newGetRequest(getURI());
+		getAllObsIncludingVoidedRequest.addParameter("encounter", "62967e68-96bb-11e0-8d6b-9b9415a91465");
+		getAllObsIncludingVoidedRequest.addParameter("includeAll", "true");
+		MockHttpServletResponse allObsIncludingVoidedResponse = handle(getAllObsIncludingVoidedRequest);
+		List<Object> allObsIncludingVoidedList = deserialize(allObsIncludingVoidedResponse).get("results");
+		
+		assertEquals(allObsIncludingVoidedList.size(), 6);
```

Comment:
The test does not assert that voided obs were retrieved, because AllNonVoidedObsList has the same size with  allObsIncludingVoidedList. Can you fix that?

Reply:
By this I was confirming that the length is the same since before the fix it would only retrieve the non-voided observations hence length would be less but with above test it confirms that even after the delete request the length is still the same.

Reply:
Ok, just assert that AllNonVoidedObsList is now 5.

Reply:
Oh, well noted

Reply:
@adamsdenniskariuki , @andela-cuchendu  I have added the assert.
Thanks for the review.